.github/workflows/ci.yml

```diff
@@ -42,7 +42,20 @@ jobs:
       uses: dtolnay/rust-toolchain@stable
 
     - name: Run cargo test
-      run: cargo test --verbose --all-features
+      run: cargo test --verbose --features __all-non-conflicting-features,abi
+
+  test-schemars-v1:
+    runs-on: ubuntu-20.04
```

Comment:
Use `ubuntu-latest` as 20.04 was deprecated, and for crates we are good to test them on latest.